Northstar.Client/mod.json

```
@@ -46,6 +46,10 @@
 			"Name": "modlist_reverse",
 			"DefaultValue": "0",
 			"Flags": "ARCHIVE_PLAYERPROFILE"
+		},
+		{
+			"Name": "menu_show_promos",
+			"DefaultValue": "1"
```

As suggested by @ASpoonPlaysGames in https://github.com/R2Northstar/NorthstarMods/pull/784#issuecomment-2243775322, this change makes the parameter persist through game reboots (tested and approved, thanks again Spoon)
```suggestion
			"DefaultValue": "1",
			"Flags": "ARCHIVE_PLAYERPROFILE"
```